### ISSUE TITLE
Addon Conflict with RareScanner

### ISSUE THREAD

**Issue by LiangYuxuan** [Jaliborc/PetTracker#150]
**Running Software (issues missing this information will be deleted):**
 - Addon version: 8.3.1
 - Server patch: Build 8.3.0.33369

 **Have you read the changelog? (please don't waste our time)**
 Yes.

**Describe the bug**
PetTracker fails to share the world map tracking dropdown menu with RareScanner, because RareScanner hooks the original menu but PetTracker replace it and build a new one.

**To Reproduce**
1. Enable RareScanner only and find dropdown menu works.
2. Enable PetTracker and find the menu replace by PetTracker, where RareScanner options are gone.

**Expected behaviour**
PetTracker and RareScanner should share the same dropdown menu and work together :)

**Screenshots**
-

**Error Logs**
No errors.

**Additional context**
You can find RareScanner's implementation in `RareScanner/MapPlugin/RsWorldMapHooks.lua`. It simply uses hooks to implement this.

**Comment by Jaliborc** [comment]
Yeah, hooking the original dropdown taints the game, I ain't doing that.

**Comment by Jaliborc** [comment]
But I have contacted the author, maybe we can find something

**Comment by arithmandar** [comment]
[World Map Tracking Enhanced](https://www.curseforge.com/wow/addons/world-map-tracking-enhanced) can be a good workaround

**Comment by fubaWoW** [comment]
> [World Map Tracking Enhanced](https://www.curseforge.com/wow/addons/world-map-tracking-enhanced) can be a good workaround

Yeah, the DropDown from PetTracker will break **every** AddOn (for years and still) that hooks the "Default" DropDown menu like RareScanner or any other addon that adds QOL features to the Original Filter menu!  
Also ElvUI Skinning is broken for the DropDown menu if PetTracker is active.  
Also PetTracker Dropdown breaks the skinning of World Map Tracking Enhanced's dropdown menu!  

**AND** overall it is **NOT** Possible to disable the ***broken*** dropdown menu...  

Sadly there is not really alternative for the "Enemy Team Ability Bar" i know, thats the only reason i sill use it ^^
